starters/next-drupal-starter/components/grid-list.js

```diff
@@ -0,0 +1,66 @@
+import { IMAGE_URL } from "../lib/constants";
+
+import Grid from "./grid";
+import Image from "next/image";
+import Link from "next/link";
+
+export default function GridList({ contentArr, contentType, multiLanguage }) {
```

Comment:
What was described for Article probably even applies more here. The `field_recipe_category` is really just a category (or tag) and it should be possible to use this grid with other category-like fields.

Reply:
I tried going for a HOC here similar to the new paginator that is BYOC (bring your own component). Open to tabling this component for later if it's still not up to snuff.